pkg/controller/stack/stack_controller_test.go

```diff
@@ -1787,8 +1780,8 @@ func TestReconcileActiveVersionsSkipCertVerify(t *testing.T) {
 		t.Fatal(fmt.Sprintf("Should be an error in the status message"))
 	}
 
-	if !strings.Contains(stackResource.Status.Versions[0].StatusMessage, "x509") {
```

Comment:
This is really the only issue that I have - we need a way to be able to test that we do or do not trust the certificate that the server presents, and that's what this test was doing.  Now it just sees that `router-ca` is not present.  

We need some way to proceed when `router-ca` is not present, and the client asked us to verify the server's certificate.  Some ideas:
1) If `router-ca` is not there, issue an error/warning message to the log and proceed without it.
2) Capture a `router-ca` and have the controller-runtime client for this test case return it when the code asks for it.

If we're going to insist on `router-ca` being there, then we should probably test what happens when someone replaces the default ingress certificate with a custom one (making sure `router-ca` now has the custom certificate). See:
https://docs.openshift.com/container-platform/4.3/authentication/certificates/replacing-default-ingress-certificate.html